tfhe/build.rs

```diff
@@ -73,7 +73,71 @@ fn gen_c_api() {
         .write_to_file(output_file);
 }
 
+fn panic_if_non_optimized_build() {
+    // This has a lot of \n and \ to control formatting precisely
+    const ERROR_MESSAGE: &str = "\
+    It seems that tfhe-rs is being compiled with insufficient optimization level.\n\
+    FHE is already slow on its own, and so compiler optimizations have to be turned on.\n\
+    \n\
+    By default rust/cargo does not enable optimizations, make sure you build/run with the \n\
+    release flag:\n    \
+    - `cargo run --release`\n    \
+    - `cargo build --release`\n\
+    (Note that doing `cargo build --release && cargo run`, will build in release mode\n\
+     but then rebuild in debug mode and run in debug mode)\n
+    \n\
+    * If you wish to be able to use a debugger, consider adding the following lines \n\
+    to the projects's Cargo.toml and compile in release\n\
+    \t```\n\
+    \t[profile.release]\n\
+    \tdebug = true\n\
```

Comment:
You could also propose to set opt-level to 3 in the debug profile

Reply:
its proposed a few lines below

Reply:
I meant the global dev profile, not the tfhe specific one